AutomatedTesting/Gem/PythonTests/Prefab/tests/PrefabTestUtils.py

```diff
@@ -190,6 +190,32 @@ def check_entity_children_count(entity_id, expected_children_count):
     return entity_children_count_matched
 
 
+def validate_count_for_editor_entity(entity_name, expected_count):
```

Comment:
Suggestion: change function names to "validate_count_for_named_editor_entity" and "validate_child_count_for_named_editor_entity" so that it's more clear what the functions are validating.

Reply:
good suggest. I like the new names.